packages/apps/graph/README.md

```diff
@@ -21,8 +21,8 @@ A GraphQL endpoint that interacts with chainweb-data.
 First, install dependencies and build up to and including @kadena/graph
 
 ```sh
-rush install -t @kadena/graph
-rush build -t @kadena/graph
+pnpm install --filter @kadena/graph
+pnpm build --filter @kadena/graph
```

Comment:
This will only install and build the ONE package.
Use the postfix `...` to also include dependencies
See: https://turbo.build/repo/docs/core-concepts/monorepos/filtering#include-dependencies-of-matched-workspaces

```suggestion
pnpm install --filter @kadena/graph...
pnpm build --filter @kadena/graph...
```